src/borg/remote.py

```diff
@@ -17,7 +17,8 @@
 from .helpers import bin_to_hex
 from .repository import Repository
 
-RPC_PROTOCOL_VERSION = 2
+RPC_PROTOCOL_VERSION = 3
+RPC_PROTOCOL_DOWNGRADABLE_VERSION = 2
```

Comment:
That's harder because the client only supports a single supported version. If that version isn't in the supported set which item in the list do we send back?